docs/apis/streaming/fault_tolerance.md

```diff
@@ -176,6 +176,11 @@ state updates) of Flink coupled with bundled sinks:
         <td></td>
     </tr>
     <tr>
+        <td>Cassandra sink</td>
+        <td>exactly once</td>
```

Comment:
I would try to summarize the guarantees here. Or maybe add an asterisk to the "exactly once", stating that Cassandra has no support for large atomic inserts, but as long as cassandra is not failing, Flink failures will not cause duplicates.